src/toolkit/gis.cc

```diff
@@ -0,0 +1,158 @@
+#include "gis.h"
+
+using namespace std;
+
+namespace cyclus {
+namespace toolkit {
+
+GIS::GIS() : latitude_(0), longitude_(0) {}
+
+GIS::GIS(float decimal_lat, float decimal_lon) {
+  latitude_ = SetPrecision(decimal_lat * 3600, 1);
+  longitude_ = SetPrecision(decimal_lon * 3600, 1);
+}
+
+GIS::~GIS() {}
+
+float GIS::get_latitude() const { return SetPrecision(latitude_ / 3600, 6); }
+
+float GIS::get_longitude() const { return SetPrecision(longitude_ / 3600, 6); }
+
+void GIS::set_latitude(float lat) { latitude_ = SetPrecision(lat * 3600, 1); }
+
+void GIS::set_longitude(float lon) { longitude_ = SetPrecision(lon * 3600, 1); }
+
+double GIS::Distance(GIS target) const {
+  double curr_longitude = this->get_longitude() * M_PI / 180;
+  double curr_latitude = this->get_latitude() * M_PI / 180;
+  double target_longitude = target.get_longitude() * M_PI / 180;
+  double target_latitude = target.get_latitude() * M_PI / 180;
+  double dlong = target_longitude - curr_longitude;
+  double dlat = target_latitude - curr_latitude;
```

Comment:
I've done this because I've read that calculations involving floating points can become inaccurate, especially, when I am multiplying by PI, or calculating the sin or cos so, I thought it would be better to use doubles here.

I'm new to programming so if these practices are poor or not recommended, please let me know. Thank you for the comments.